assignments/stretch-function-conversion.js

```diff
@@ -1,23 +1,30 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
 // let myFunction = function () {};
-
+let myFunction = () => {}
 // let anotherFunction = function (param) {
 //   return param;
 // };
-
+let anotherFunction = (param) => param
```

Comment:
The parens are not required if there is only one parameter. `let anotherFunction = param => param` is perfectly valid. Some people do like to add the parens anyway because

1. It distinguishes the parameter from the return value clearly.
2. It is convenient when you want to add more params later, at which point the parens are required.